src/Problems/PBO/f_ising_triangle.hpp

```diff
@@ -39,19 +35,27 @@ class Ising_Triangle : public IOHprofiler_problem<int> {
     int i,j,neig;
     int n = x.size();
     int result = 0;
-    int neighbors[6];
+    // int neighbors[6];
+    int neighbors[3];
     int lattice_size = (int)sqrt((double)n);
     
     for (i = 0; i < lattice_size; ++i) {
       for (j = 0; j < lattice_size; ++j) {
-        neighbors[0] = x[modulo_ising_triangle((i - 1), lattice_size) * lattice_size + j] ;
-                neighbors[1] = x[modulo_ising_triangle((i + 1), lattice_size) * lattice_size + j] ;
-                neighbors[2] = x[i * lattice_size + modulo_ising_triangle((j - 1) , lattice_size)] ;
-                neighbors[3] = x[i * lattice_size + modulo_ising_triangle((i + 1) , lattice_size)] ;
-                neighbors[4] = x[modulo_ising_triangle((i - 1) , lattice_size) * lattice_size + modulo_ising_triangle((j - 1) , lattice_size)] ;
-                neighbors[5] = x[modulo_ising_triangle((i + 1) , lattice_size) * lattice_size + modulo_ising_triangle((j + 1) , lattice_size)];
+        // neighbors[0] = x[modulo_ising_triangle((i - 1), lattice_size) * lattice_size + j] ;
+        // neighbors[1] = x[modulo_ising_triangle((i + 1), lattice_size) * lattice_size + j] ;
+        // neighbors[2] = x[i * lattice_size + modulo_ising_triangle((j - 1) , lattice_size)] ;
+        // neighbors[3] = x[i * lattice_size + modulo_ising_triangle((i + 1) , lattice_size)] ;
+        // neighbors[4] = x[modulo_ising_triangle((i - 1) , lattice_size) * lattice_size + modulo_ising_triangle((j - 1) , lattice_size)] ;
+        // neighbors[5] = x[modulo_ising_triangle((i + 1) , lattice_size) * lattice_size + modulo_ising_triangle((j + 1) , lattice_size)];
+
+        neighbors[0] = x[modulo_ising_triangle((i + 1), lattice_size) * lattice_size + j] ;
+        neighbors[1] = x[i * lattice_size + modulo_ising_triangle((i + 1) , lattice_size)] ;
```

Comment:
should this be `modulo_ising_triangle((j + 1) , lattice_size)`? You are retrieving the neighbour on row `i` and column `j+1`, right?

Reply:
Yes. You are right.